src/Microsoft.Windows.CsWin32/Generator.cs

```diff
@@ -4102,7 +4105,8 @@ private IEnumerable<MethodDeclarationSyntax> DeclareFriendlyOverloads(MethodDefi
                 }
                 else if ((externParam.Type is PointerTypeSyntax { ElementType: TypeSyntax ptrElementType }
                     && !IsVoid(ptrElementType)
-                    && !this.IsInterface(parameterTypeInfo)) ||
+                    && !this.IsInterface(parameterTypeInfo)
+                    && this.canUseSpan) ||
```

Comment:
My confidence is rather low in this placement and the following one. Does this make sense to you or did I leave holes?

Reply:
I'm not sure either. We can take it as-is and fix it with more tests if anything comes up. It seems safe enough since `canUseSpan` is true in the officially supported scenarios, so it seems at worst your change would simply not cut out use of span in enough places.